github/search.go

```diff
@@ -54,6 +55,37 @@ func (s *SearchService) Repositories(query string, opt *SearchOptions) (*Reposit
 	return result, resp, err
 }
 
+// CommitsSearchResult represents the result of a commits search.
+type CommitsSearchResult struct {
+	Total             *int           `json:"total_count,omitempty"`
```

Comment:
In a normal situation, I would full you agree.

However, please look at the other similar structs in this file @gmlewis. See `RepositoriesSearchResult`, `CodeSearchResult`, etc.

They consistently call total_count as Total.